app/models/user.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define(
+    'user',
+    {
+      name: { allowNull: false, type: DataTypes.STRING },
+      last_name: { allowNull: false, type: DataTypes.STRING },
```

Comment:
The properties in the model go in `camelCase`, there you must find the way to convert them to `snake_case`.
Hint: Look for a property that is added to the model for this.